templates/docs/examples/layouts/application.html

```diff
@@ -19,7 +19,7 @@
             <div class="p-inline-list--stretch">
               <div class="p-inline-list__item">
                 <a class="p-panel__logo" href="#">
-                  <img src="https://assets.ubuntu.com/v1/0b2b7bce-Vanilla-logo_88x24px_white.svg" alt="Vanilla framework logo" height="24">
```

Comment:
Shouldn't this one be "Vanilla framework"? It seems to be the same as 'home page logo' link in main nav.